app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -149,8 +149,9 @@ def find_host_by_provider_id(provider_id)
     def scheme_to_provider_mapping
       @scheme_to_provider_mapping ||= begin
         {
-          'gce' => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
-          'aws' => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'gce'   => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
+          'aws'   => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'azure' => ['ManageIQ::Providers::Azure::CloudManager'.safe_constantize, :uid_ems],
```

Comment:
@enoodle right, I see that it calls the method on Vm model. In this case :uid_ems and :ems_ref should be the same (if you check the refresh code of those providers, grep for methods 'parse_instance').

But I would prefer :ems_ref, since that is the remote UUID everywhere